include/zephyr/bluetooth/iso.h

```diff
@@ -860,14 +872,12 @@ int bt_iso_chan_disconnect(struct bt_iso_chan *chan);
  *                  interval for a specific channel.
  *  @param ts       Timestamp of the SDU in microseconds (us).
  *                  This value can be used to transmit multiple
- *                  SDUs in the same SDU interval in a CIG or BIG. Can be
- *                  omitted by using @ref BT_ISO_TIMESTAMP_NONE which will
- *                  simply enqueue the ISO SDU in a FIFO manner.
+ *                  SDUs in the same SDU interval in a CIG or BIG.
  *
  *  @return Bytes sent in case of success or negative value in case of error.
  */
-int bt_iso_chan_send(struct bt_iso_chan *chan, struct net_buf *buf,
-		     uint16_t seq_num, uint32_t ts);
+int bt_iso_chan_send_ts(struct bt_iso_chan *chan, struct net_buf *buf, uint16_t seq_num,
```

Comment:
Hmm, I see your point, although it seems weird to mention what is omitted in the function name. This would be similar to having a negated flag - for example the HCI ISO data packet sets a flag if the TS is included. Your suggestion would be similar to setting a flag when it was not included. 

I suggest that we keep it as is for 2 reasons:
1) None of our samples or tests have been using TS, so it's clearly not widely used (and not the default)
2) Omitting `ts` from the non-TS version is more aligned with the core spec in terms of setting the flag when it is there